about.html

```diff
@@ -79,7 +82,7 @@ <h2>Digitalization In Medicine Forum Logo</h2>
 
         <div id="div31">
```

Comment:
- [ ] **The design for the past medicine section can be improved to match the original design. Please consider it**

Original Version|Cloned Version
:--------------------:|:------------------:
![image](https://user-images.githubusercontent.com/59168430/125165709-c7a8f100-e1b5-11eb-971c-07ccab349bc0.png)|![image](https://user-images.githubusercontent.com/59168430/125165697-bbbd2f00-e1b5-11eb-8782-0915f94e2884.png)

The original design contains images, please consider adding some placeholder images. Also, the images are more in landscape orientation in the original design.